source/fuzz/transformation_context.cpp

```diff
@@ -18,10 +18,8 @@ namespace spvtools {
 namespace fuzz {
 
 TransformationContext::TransformationContext(
-    FactManager* transformation_context,
-    spv_validator_options validator_options)
-    : fact_manager_(transformation_context),
-      validator_options_(validator_options) {}
+    FactManager* fact_manager, spv_validator_options validator_options)
+    : fact_manager_(fact_manager), validator_options_(validator_options) {}
```

Comment:
Weird - did clang-format require this change?

Reply:
No, I just noticed that `FactManager` argument had a wrong name (its name was `transformation_context`) so I decided to rename it.

Reply:
Ah good - thanks!